docs/policies_and_procedures/index.md

```diff
@@ -12,4 +12,3 @@ data_license
 cruise_documentation
 data_evaluation_reference
 woce_era_manuals
----
```

Comment:
This PR didn't do this, but we are missing the closing triple back-tics for many of the toctree directives. We'll need to fix that in a followup PR I think.